website/docs/guides/orchestration/custom-cicd-pipelines/1-creating-custom-cicd-pipelines.md

```diff
@@ -0,0 +1,431 @@
+
+```yaml
+my_awesome_project
+├── python
+│   └── run_and_monitor_dbt_job.py
+```
+
+This Python file has everything you need to call the dbt Cloud API, but requires a few inputs (see snip below). Those inputs are fed to this script through environment variables that will be defined in the next step.
+
+```python
+#------------------------------------------------------------------------------
+# get environment variables
+#------------------------------------------------------------------------------
+api_base        = os.getenv('DBT_URL', 'https://cloud.getdbt.com/') # default to multitenant url
+job_cause       = os.getenv('DBT_JOB_CAUSE', 'API-triggered job') # default to generic message
+git_branch      = os.getenv('DBT_JOB_BRANCH', None) # default to None
+schema_override = os.getenv('DBT_JOB_SCHEMA_OVERRIDE', None) # default to None
+api_key         = os.environ['DBT_API_KEY']  # no default here, just throw an error here if key not provided
+account_id      = os.environ['DBT_ACCOUNT_ID'] # no default here, just throw an error here if id not provided
+project_id      = os.environ['DBT_PROJECT_ID'] # no default here, just throw an error here if id not provided
+job_id          = os.environ['DBT_PR_JOB_ID'] # no default here, just throw an error here if id not provided
+```
+
+**Required input:**
+
+In order to call the dbt Cloud API, there are a few pieces of info the script needs. The easiest way to get these values is to open up the job you want to run in dbt Cloud. The URL when you’re inside the job has all the values you need:
+
+- `DBT_ACCOUNT_ID` - this is the number just after `accounts/` in the URL
+- `DBT_PROJECT_ID` - this is the number just after `projects/` in the URL
+- `DBT_PR_JOB_ID` - this is the number just after `jobs/` in the URL
+
+![Image of a dbt Cloud job URL with the pieces for account, project, and job highlighted](/img/guides/orchestration/custom-cicd-pipelines/dbt-cloud-job-url.png)
+
+
+
+### 4. Update your project to include the new API call
+
+<details>
+<summary> GitHub </summary>
+    
+For this new job, we’ll add a file for the dbt Cloud API call named `dbt_run_on_merge.yml`.
+
+```yaml
+my_awesome_project
+├── python
+│   └── run_and_monitor_dbt_job.py
+├── .github
+│   ├── workflows
+│   │   └── dbt_run_on_merge.yml
+│   │   └── lint_on_push.yml
+```
+
+The yaml file will look pretty similar to our earlier job, but there is a new section called `env` that we’ll use to pass in the required variables. Update the variables below to match your setup based on the comments in the file.
+
+Since these are so important, it’s worth noting that we changed the `on:` section to now run **only** when there are pushes to a branch named `main` (i.e. a PR is merge). Have a look through [GitHub’s docs](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows) on these filters for additional use cases.
```

Comment:
"Since these are so important" is a bit vague - I think we can just delete this and go directly into "it’s worth noting that we changed"

Reply:
I like it, changed :)